web/templates/recipient_row.hbs

```diff
@@ -100,6 +100,9 @@
             {{/inline}}
             {{/tr~}}
         </span>
+        {{#each recipient_users}}
+            {{#if is_bot}}<i class="zulip-icon zulip-icon-bot" aria-label="{{t 'Bot' }}"></i>{{/if}}
+        {{/each}}
```

Comment:
I don't see how this being a loop can possibly make sense -- this needs to be inside the loop above, or you'll just get 3 bot icons in a row.

Reply:
Yep, it's bugged.

![image](https://github.com/user-attachments/assets/ca3aa8d4-e393-44b9-96d5-8e677ac5a97d)

Reply:
I dropped this commit, since I think it requires some added work to finish. @NotVishesh can you put up an updated PR with just that commit? I'm going to merge this one.